src/components/Link.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link as USWDSLink } from "@trussworks/react-uswds";
+import { Link as GatsbyLink } from "gatsby";
+
+// Pass only activeClassName and partiallyActive to GatsbyLink
+// Pass only fileType and size to file downloads
+const Link = ({
+  children,
+  to,
+  activeClassName,
+  partiallyActive,
+  variant,
+  fileType,
+  size,
+  ...other
+}) => {
+  // Assumes that any internal link will start with exactly one slash, and that anything else is external.
+  const internal = /^\/(?!\/)/.test(to);
```

Comment:
You could also check for different domain name?

Reply:
By satisfying this regex, it satisfies that it's in the same domain name. It checks to see if the entered link is a route. If the entered link is a route, we can assume it's still on the same domain.